felles/oidc/src/main/java/no/nav/vedtak/sikkerhet/oidc/config/impl/OidcProviderConfig.java

```diff
@@ -26,24 +25,23 @@ public final class OidcProviderConfig {
     private static final Environment ENV = Environment.current();
     private static final Logger LOG = LoggerFactory.getLogger(OidcProviderConfig.class);
 
-    public static final String OPEN_AM_WELL_KNOWN_URL = "oidc.open.am.well.known.url";
+    private static final String OPEN_AM_WELL_KNOWN_URL = "oidc.open.am.well.known.url";
     private static final String OPEN_AM_CONFIG_ISSUER = "oidc.open.am.openid.config.issuer";
     private static final String OPEN_AM_CONFIG_JWKS_URI = "oidc.open.am.openid.config.jwks.uri";
     private static final String OPEN_AM_CONFIG_TOKEN_ENDPOINT = "oidc.open.am.openid.config.token.endpoint";
-    private static final String OPEN_AM_CONFIG_AUTH_ENDPOINT = "oidc.open.am.openid.config.authorization.endpoint";
-    public static final String OPEN_AM_CLIENT_ID = "oidc.open.am.client.id";
-    public static final String OPEN_AM_CLIENT_SECRET = "oidc.open.am.client.secret";
+    private static final String OPEN_AM_CLIENT_ID = "oidc.open.am.client.id";
+    private static final String OPEN_AM_CLIENT_SECRET = "oidc.open.am.client.secret";
 
     private static final String STS_WELL_KNOWN_URL = "oidc.sts.well.known.url";
     private static final String STS_CONFIG_ISSUER = "oidc.sts.openid.config.issuer";
     private static final String STS_CONFIG_JWKS_URI = "oidc.sts.openid.config.jwks.uri";
     private static final String STS_CONFIG_TOKEN_ENDPOINT = "oidc.sts.openid.config.token.endpoint";
 
     private static final String AZURE_WELL_KNOWN_URL = "azure.app.well.known.url"; // naiserator
-    private static final String AZURE_CONFIG_ISSUER = "azure.openid.config.issuer"; // naiserator
-    private static final String AZURE_CONFIG_JWKS_URI = "azure.openid.config.jwks.uri"; // naiserator
+    public static final String AZURE_CONFIG_ISSUER = "azure.openid.config.issuer"; // naiserator
```

Comment:
Eh, er det ikke bedre å kapsle inn internals. Hva trenger du puclic til?

Reply:
Tror vi trenger disse for validering?

Reply:
Skal gå gjennom og fjerne alt grundig, så kan vi fjerne det helt. Jeg gikk ut at vi fortsatt må kunne validere tokens fra ISSO. Men om de skal være så streng og slå av ISSO fra 1/1 så er det ikke noe vits å ha dette inne i koden.